services/ui-src/src/components/fields/TextAreaField.test.tsx

```diff
@@ -72,6 +76,40 @@ describe("<TextAreaField />", () => {
     });
   });
 
+  describe("Text area field hide condition logic", () => {
+    test("Text area field is hidden if its hide conditions' controlling element has a matching answer", async () => {
+      mockGetValues({
+        elements: [
+          {
+            answer: "yes",
+            type: "reportingRadio",
+            label: "Should we hide the other radios on this page?",
+            id: "reporting-radio",
+          },
+        ],
+      });
+      render(textAreaFieldComponent);
+      const textField = screen.queryByLabelText("test label");
+      expect(textField).toBe(null);
```

Comment:
Suggestion: You're already using a more semantic assertion (`toBeVisible`) for the opposite case below; invert it here to be equally semantic.
```suggestion
      expect(textField).not.toBeVisible();
```

Reply:
i would love to do this, but if the element doesn't exist on the page, it refuses to let me call .not.toBeVisible and the error is this:
    
```
    expect(received).not.toBeVisible()

    received value must be an HTMLElement or an SVGElement.
    Received has value: null
```

Do you know how to get around that? I assume it's because I'm querying (`queryByLabelText` instead of `getByLabelText`) so the query returns null if the element doesn't exist. but if do getByLabelText, it just can't find the element and fails with this error:

`TestingLibraryElementError: Unable to find a label with the text of: test label`

Reply:
🤦 You are totally right, that is my mistake.

Reply:
Ok I updated to use .not.toBeInTheDocument. Thank you for this feedback!